### PR TITLE
CO-2648_Password_Dashboard_Widget

### PR DESCRIPTION
**Pull request by ioigoume**

### CODE REVIEW

File: app/webroot/css/co-responsive.css [hunk 1]
```diff
@@ -533,7 +533,6 @@
     align-items: center;
```
Comment (spartanicus) [inline app/webroot/css/co-responsive.css:533]: this can go

File: app/webroot/css/co-responsive.css [hunk 2]
```diff
@@ -533,7 +533,6 @@
...
     border-bottom: 1em solid transparent;
     justify-content: space-between;
-    align-items: end;
```
Comment (spartanicus) [inline app/webroot/css/co-responsive.css:536]: this should stay

File: app/webroot/css/co-base.css [hunk 3]
```diff
@@ -2816,8 +2816,8 @@ ul.cm-ssw-field-list ul.dropdown-menu button.btn {
 }
 .cm-ssw-form-row-fields {
   display: flex;
-  align-items: end;
```
[the remark this answers is not in the document]
Comment (ioigoume) [reply app/webroot/css/co-base.css:2819]: done

File: app/webroot/css/co-responsive.css (hunk 2 above)
Comment (ioigoume) [reply app/webroot/css/co-responsive.css]: done

File: app/webroot/css/co-base.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (spartanicus) [reply app/webroot/css/co-base.css:2819]: bah - I was wrong about this one. Pull out the "align-items: end" on it. I remember now why it was pulled. (Sorry for the inconsistency.)

[the remark this answers is not in the document]
Comment (ioigoume) [reply app/webroot/css/co-base.css:2819]: done

**Pull request merged**